spec/models/candidate_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe Candidate, type: :model do
+  it "has a valid factory" do
+    expect(build(:candidate)).to be_valid
+  end
+
+  # Lazily loaded to ensure it's only used when it's needed
+  let(:cand) { build(:candidate) }
+  let(:saved) { create(:candidate) }
+
+  describe "ActiveModel validations" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.